src/preamble.js

```diff
@@ -165,7 +165,7 @@ var cwrap, ccall;
   var toC = {'string' : JSfuncs['stringToC'], 'array' : JSfuncs['arrayToC']};
 
   // C calling interface.
-  ccall = function ccallFunc(ident, returnType, argTypes, args, opts) {
```

Comment:
why remove the function names?

Reply:
actually, i think since after this change we have just
````
var ccall;
ccall = function ..
````
(instead of two possible assigns to `ccall`), we can just move to
````
function ccall ..
````

Reply:
These functions are wrapped in IIFE, so if they are given some name - it will not be available from the outside. This is the reason I've removed function names - they can't be called from anywhere anyway.

Reply:
I don't really see much need for IIFE there though, unwrapped it in latest commit